scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -48,12 +54,14 @@ namespace <%= namespace %>.SDK {
         public string ID = null;
         private string _VariantId;
         private long _Quantity;
+        private decimal _Price;
         private ObservableDictionary<string, string> _CustomAttributes;
         private OnCartLineItemChange OnChange;
-        
-        public CartLineItem(string variantId, OnCartLineItemChange onChange, long quantity = 1, IDictionary<string, string> customAttributes = null) {
+
+        public CartLineItem(string variantId, OnCartLineItemChange onChange, decimal price, long quantity = 1, IDictionary<string, string> customAttributes = null) {
```

Comment:
This feels kind of shaky since we give power to the user to enter their own price, which is always a bad idea. What are the implications if instead `CartLineItem` takes in a `ProductVariant` instead of `variantID` and `price`. That way `CartLineItem` can get the variant ID, price, and any other variant info down the road